README.rst

```diff
@@ -340,206 +342,202 @@ For example:
+For details, see :ref:`build and packaging files <cartridge-cli-special-files>`.
 
-.. _cartridge-cli-starting-stopping-an-application-locally:
+As a result, a fully built application will appear in the ``.rocks`` directory.
+You can start it locally from the application directory.
 
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-Starting/stopping an application locally
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. _cartridge-cli-starting-the-application-locally:
 
-**********
-``start``
-**********
+Starting the application locally
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Now, after the application is `built <Building an application_>`_,
+After you've :ref:`built your application <cartridge-cli-building-the-application>`,
 you can run it locally:
 
 .. code-block:: bash
 
     cartridge start [INSTANCE_NAME...] [flags]
 
-where ``[INSTANCE_NAME...]`` means that several instances can be specified.
+where ``[INSTANCE_NAME...]`` stands for one or multiple instance names.
 
-If no ``INSTANCE_NAME`` is provided, all the instances from the
-Cartridge instances configuration file are taken as arguments (see the ``--cfg``
+If no ``INSTANCE_NAME`` is provided, all the instances in the
+Cartridge instances configuration file will be taken as arguments (see the ``--cfg``
 option below).
 
-We also need an application name (``APP_NAME``) to pass it to the instances while
-started and to define paths to the instance files (for example, ``<run-dir>/<APP_NAME>.<INSTANCE_NAME>.pid``).
-By default, the ``APP_NAME`` is taken from the application rockspec in the current
-directory, but also it can be defined explicitly via the ``--name`` option
-(see description below).
+.. // TODO add links on every "below"
 
-^^^^^^^^
-Options
-^^^^^^^^
-
-The following options (``[flags]``) are supported:
-
-.. // Please, update the doc in cli/commands on updating this section
-
-* ``--script FILE`` is the application's entry point.
-  It should be a relative path to the entry point in the project directory
-  or an absolute path.
-  Defaults to ``init.lua`` (or to the value of the "script"
-  parameter in the Cartridge `configuration file <Overriding default options_>`_).
-
-* ``--run-dir DIR`` is the directory where PID and socket files are stored.
-  Defaults to ``./tmp/run`` (or to the value of the "run-dir"
-  parameter in the Cartridge `configuration file <Overriding default options_>`_).
-
-* ``--data-dir DIR`` is the directory where instances' data is stored.
-  Each instance's working directory is ``<data-dir>/<app-name>.<instance-name>``.
-  Defaults to ``./tmp/data`` (or to the value of the "data-dir"
-  parameter in the Cartridge `configuration file <Overriding default options_>`_).
+The application name, ``APP_NAME``, is passed to the instances during startup and
+used in instance file paths,
+for example: ``<run-dir>/<APP_NAME>.<INSTANCE_NAME>.pid``).
```

Comment:
Whether the `<run-dir>` is the current directory or some another specially created directory? AFAIS, it is not mentioned anywhere above.

Reply:
Ah, I see. It is below. Not nice, but at least does not incorrect.